client/network/src/transport.rs

```diff
@@ -145,5 +151,6 @@ pub fn build_transport(
 			.map_err(|err| io::Error::new(io::ErrorKind::Other, err))
 			.boxed();
 
-	(transport, sinks)
+	(transport, bandwidth)
 }
+
```

Comment:
We don't end files with a newline in Substrate, right?

Reply:
I don't think there's a rule about this

Reply:
I don't know, this is not intentional, probably just happened as a result of moving code around. I don't pay much attention to such syntactic details. Should it be removed?

Reply:
Thought we had this as a rule. I don't have a strong opinion. As you like.